mars/dataframe/sort/sort_values.py

```diff
@@ -0,0 +1,254 @@
+                                                  _na_position=na_position,
+                                                  _ignore_index=ignore_index,
+                                                  _parallel_kind=parallel_kind,
+                                                  _psrs_kinds=psrs_kinds,
+                                                  _object_type=ObjectType.dataframe, **kw)
+
+    @property
+    def by(self):
+        return self._by
+
+    @property
+    def axis(self):
+        return self._axis
+
+    @property
+    def ascending(self):
+        return self._ascending
+
+    @property
+    def inplace(self):
+        return self._inplace
+
+    @property
+    def kind(self):
+        return self._kind
+
+    @property
+    def na_position(self):
+        return self._na_position
+
+    @property
+    def ignore_index(self):
+        return self._ignore_index
+
+    @property
+    def parallel_kind(self):
+        return self._parallel_kind
+
+    @property
+    def psrs_kinds(self):
+        return self._psrs_kinds
+
+    @classmethod
+    def _tile_psrs(cls, op, in_data):
+        out = op.outputs[0]
+        in_df, axis_chunk_shape, _, _ = cls.preprocess(op, in_data=in_data)
+
+        # stage 1: local sort and regular samples collected
+        sorted_chunks, _, sampled_chunks = cls.local_sort_and_regular_sample(
+            op, in_df, axis_chunk_shape, None, None)
+
+        # stage 2: gather and merge samples, choose and broadcast p-1 pivots
+        concat_pivot_chunk = cls.concat_and_pivot(
+            op, axis_chunk_shape, (), sorted_chunks, sampled_chunks)
+
+        # stage 3: Local data is partitioned
+        partition_chunks = cls.partition_local_data(
+            op, axis_chunk_shape, sorted_chunks, None, concat_pivot_chunk)
+
+        proxy_chunk = DataFrameShuffleProxy(object_type=ObjectType.dataframe).new_chunk(
+            partition_chunks, shape=())
+
+        # stage 4: all *ith* classes are gathered and merged
+        partition_sort_chunks = cls.partition_merge_data(
+            op, False, None, partition_chunks, proxy_chunk)[0]
+
+        if op.ignore_index:
+            chunks = standardize_range_index(partition_sort_chunks, axis=op.axis)
+        else:
+            chunks = partition_sort_chunks
+
+        if op.axis == 0:
+            nsplits = ((np.nan,) * len(chunks), (out.shape[1],))
+        else:
+            nsplits = ((out.shape[0],), (np.nan,) * len(chunks))
+        new_op = op.copy()
+        return new_op.new_dataframes(op.inputs, shape=out.shape, chunks=chunks,
+                                     nsplits=nsplits, index_value=out.index_value,
+                                     columns_value=out.columns_value, dtypes=out.dtypes)
+
+    @classmethod
+    def tile(cls, op):
+        df = op.inputs[0]
+
+        if op.axis == 0:
+            df = build_concatenated_rows_frame(df)
+        else:
+            df = df.rechunk({0: (df.shape[0],)})._inplace_tile()
```

Comment:
I think for now we may drop support for axis == 1 because it generates unknown columns.